_sass/base/components/_picc-slider.scss

```diff
@@ -0,0 +1,60 @@
+picc-slider {
+  $handle-size: 24px;
+  $horizontal-margin: $handle-size * 1.5;
+  $label-width: 80px;
+  $mark-outset: 4px;
+  $slider-height: 6px;
+  $vertical-margin: 2.5em;
+  $vertical-offset: ($slider-height - $handle-size) / 2;
+
+  background: #ccc;
+  display: block;
+  height: $slider-height;
+  margin: $vertical-margin $horizontal-margin;
+  position: relative;
+
+  .label {
```

Comment:
This class should be namespaced if this is the only place it will ever live: `.picc-slider-label`. This will help us debug code faster (it will be clear where the code is) and also help prevent accidental overlap in names/style application.